tests/foreman/ui/test_contenthost.py

```diff
@@ -1313,3 +1315,39 @@ def test_pagination_multiple_hosts_multiple_pages(session, module_host_template)
         # Assert that total items reported is the number of hosts created for this test
         total_items_found = pagination_values['total_items']
         assert int(total_items_found) >= host_num
+
+
+@tier3
+def test_search_for_virt_who_hypervisors(session):
+    """
+    Search the virt_who hypervisors with hypervisor=True or hypervisor=False.
+
+    :id: 3c759e13-d5ef-4273-8e64-2cc8ed9099af
+
+    :expectedresults: Search with hypervisor=True and hypervisor=False gives the correct result.
+
+    :BZ: 1653386
+
+    :CaseLevel: System
+
+    :CaseImportance: Medium
+    """
+    org = entities.Organization().create()
+    with session:
+        session.organization.select(org.name)
```

Comment:
no need to create and select org separately. give it a try and see if it works for you :)

Reply:
Hey, Jameer, I saw the `module_org` function, but the scope of it is `module`, which means it will create org at the start. I create a new org separately to avoid some remaining hypervisors from other test cases. It may affect our test results. What do you think?

Reply:
got it. :+1: